index.js

```diff
@@ -65,14 +65,12 @@ const ERROR_MESSAGE_PARSING_FAILED = 'ps output parsing failed';
 
 const psFields = 'pid,ppid,uid,%cpu,%mem,comm,args';
 
-// TODO: Use named capture groups when targeting Node.js 10
-const psOutputRegex = /^[ \t]*(?<pid>\d+)[ \t]+(?<ppid>\d+)[ \t]+(?<uid>\d+)[ \t]+(?<cpu>\d+\.\d+)[ \t]+(?<memory>\d+\.\d+)[ \t]+/;
+const psOutputRegex = /^[\s]*(?<pid>\d+)[\s]+(?<ppid>\d+)[\s]+(?<uid>\d+)[\s]+(?<cpu>\d+\.\d+)[\s]+(?<memory>\d+\.\d+)[\s]+(?<commAndArgs>.*)/;
```

Comment:
This regex is getting quite verbose now. I think it could benefit from being exported with an underscore prefix and then tested against various static output strings, to ensure it handles all cases

Reply:
This regex is fully tested in my [php package](https://github.com/webdevium/processes/blob/master/src/Processes.php#L46)

But for busybox we need another regex, anyway